packages/i18n/src/locales/en_US.ts

```diff
@@ -2045,6 +2045,30 @@ const translations: Catalog = {
                 'Returns a cursor with information about metadata inconsistencies',
               example: 'sh.checkMetadataConsistency(<options>)',
             },
+            moveCollection: {
+              link: 'https://docs.mongodb.com/manual/reference/method/sh.moveCollection',
```

Comment:
The current link to this helper (moveCollection) is https://www.mongodb.com/docs/upcoming/reference/method/sh.moveCollection/#mongodb-method-sh.moveCollection and i've added is to the future one, follows the same pattern as other existing commands here. 
For the other 3 helpers, there exists no link in upcoming docs and i anyway put it here. 

If this does not feel ok, I'd drop it.